nixos/modules/programs/chromium.nix

```diff
@@ -19,6 +19,8 @@ in
     programs.chromium = {
       enable = lib.mkEnableOption "{command}`chromium` policies";
```

Comment:
That's changing the behaviour of this module, so this option should also have been updated.

Reply:
Good catch. I'll submit a PR fixing this.

Reply:
https://github.com/NixOS/nixpkgs/pull/334012